guessTheNumber.js

```diff
@@ -1,17 +1,42 @@
+const readline = require('readline');
 // ### 1: Guess The Number
 //
 // Write a program that will generate a number between 1 and 100. Your program will ask the user to guess the generated number.
 //
 // When the user types a guess, the program should tell them if they won (the guess matched the computer's number), if their guess was higher than the number the computer generated, OR if the guess was lower than the number the computer generated. Terminate the program if the user types "exit".
 
-  var compNum = []
+const rl = readline.createInterface({
```

Comment:
I try to avoid the use of abbreviations, to avoid the possible misunderstanding of what a variable is or does.  As well, I prefer the use of pronounceable variable names to facilitate communication about my code.